Description: Fix updated for empty pubDate
 Make the test suite pass, since feedparser bahaviour has changed.
Author: Olivier Berger <obergix@debian.org>

--- planet-venus-0~git9de2109.orig/planet/reconstitute.py
+++ planet-venus-0~git9de2109/planet/reconstitute.py
@@ -361,7 +361,8 @@ def reconstitute(feed, entry):
 def entry_updated(feed, entry, default = None):
     chks = ((entry, 'updated_parsed'),
             (entry, 'published_parsed'),
-            (feed,  'updated_parsed'),)
+            (feed,  'updated_parsed'),
+            (feed, 'published_parsed'),)
     for node, field in chks:
         if node.has_key(field) and node[field]:
             return node[field]
